cpp/perspective/CMakeLists.txt

```diff
@@ -67,16 +67,20 @@ function (psp_build_dep name cmake_file)
 
 	if(${name} STREQUAL arrow)
 		# Overwrite arrow's CMakeLists with our custom, minimal CMakeLists.
-		configure_file(${PSP_CMAKE_MODULE_PATH}/arrow/CMakeLists.txt ${CMAKE_BINARY_DIR}/arrow-src/cpp/ COPYONLY)
-		configure_file(${PSP_CMAKE_MODULE_PATH}/arrow/config.h ${CMAKE_BINARY_DIR}/arrow-src/cpp/src/arrow/util/ COPYONLY)
-		add_subdirectory(${CMAKE_BINARY_DIR}/arrow-src/cpp/
-			${CMAKE_BINARY_DIR}/arrow-build
+		configure_file(${PSP_CMAKE_MODULE_PATH}/${name}/CMakeLists.txt ${CMAKE_BINARY_DIR}/${name}-src/cpp/ COPYONLY)
+		configure_file(${PSP_CMAKE_MODULE_PATH}/${name}/config.h ${CMAKE_BINARY_DIR}/${name}-src/cpp/src/arrow/util/ COPYONLY)
+		add_subdirectory(${CMAKE_BINARY_DIR}/${name}-src/cpp/
+			${CMAKE_BINARY_DIR}/${name}-build
 			EXCLUDE_FROM_ALL)
 
-		include_directories(${CMAKE_BINARY_DIR}/arrow-src/cpp/src/)
+		include_directories(${CMAKE_BINARY_DIR}/${name}-src/cpp/src/)
 	elseif(${name} STREQUAL exprtk)
 		# no cmakelists - just include the header
 		include_directories(${CMAKE_BINARY_DIR}/${name}-src)
+    elseif(${name} STREQUAL exprtk)
```

Comment:
This looks like it’s replacing the CMakelists for exprtk (which doesn’t have a CMakeLists) instead of RE2?

Reply:
good catch